packages/schematics/src/collection/ngrx/index.ts

```diff
@@ -27,6 +27,10 @@ function addImportsToModule(name: string, options: Schema): Rule {
       return host;
     }
 
+    if (!options.directory) {
+      options.directory = '+state';
```

Comment:
is this required? I thought `"default": "+state",` should do the same?

Reply:
Didn't realize the `schema.json` actually handled that. Should've tested it. My b.